std/node/util.ts

```diff
@@ -4,6 +4,15 @@ import * as types from "./_util/_util_types.ts";
 
 export { types };
 
+export function inspect(object: unknown, ...opts: any) {
```

Comment:
> secondly, how come Deno uses unknown here instead of any?

Linter disallows the use of any, unknown is used in place as it is less forgiving than any.

Reply:
cool, thanks for that, never new about that functionality, makes sense.

following up the `opts: any` instead of `opts: InspectOptions`, and the earlier bug at:
https://github.com/denoland/deno/issues/7099